.circleci/config.yml

```diff
@@ -449,14 +460,43 @@ jobs:
   prime-calypso-live:
     docker:
     - image: buildpack-deps
+    working_directory: ~/wp-calypso
     steps:
+      - prepare
       - run:
           name: Prime calypso.live build
-          command: |
-            set +o errexit
-            set -o nounset
-            curl --silent "https://hash-${CIRCLE_SHA1}.calypso.live" > /dev/null
-            exit 0
+          command: ./test/e2e/scripts/wait-for-running-branch.sh
+
+  test-e2e:
+      <<: *defaults
+      working_directory: ~/wp-calypso/test/e2e
+      parallelism: 2
+      steps:
+        - prepare
```

Comment:
who in here is responsible for splitting tests between the two containers?

Reply:
We run desktop in one container and mobile in another

Reply:
Does that happen by checking the CircleCI node env variable somewhere in a script?

If there's such a clear split, should we make e2e a workflow that includes independent desktop and mobile jobs?

Reply:
> Does that happen by checking the CircleCI node env variable somewhere in a script?

Yes, there is a script that checks which CircleCI node index it is running on and runs the desktop or mobile tests based on that value.

>If there's such a clear split, should we make e2e a workflow that includes independent desktop and mobile jobs?
That is certainly an option. I am open to whatever makes things more efficient and more clear for everyone.

Reply:
It seems to me like that would be clearer, but it's not something that needs to happen in this PR.

Reply:
Yeah, I'll do that as a part of the task to add the triggers.